nodes/node_registry.go

```diff
@@ -116,19 +124,28 @@ func newRegistryEntry(nodeKindNames []string, initFunction Initializer,
 	}
 }
 
+// PlatformAttrs contains the platform attributes this node/platform is known to have in different libraries and tools.
+// Most often just the platform/provider name.
+type PlatformAttrs struct {
+	ScrapliPlatformName string
+}
+
```

Comment:
@danlindow I think to fix the issue of napalm having a different platform naming scheme, we need to add `NapalmPlatformName` here, and populate it with the relevant const in the systems that are supported by napalm.

Then to drive which platform names are used we can set the env var `CLAB_PLATFORM_NAME_SCHEMA: napalm` to select what values to use in the templating.

Feel free to take a stab at it, or I can have a look at it sometime later

Reply:
I'll take a shot at it! Thanks for the help!